main.nf

```diff
@@ -138,7 +138,7 @@ if (!(workflow.runName ==~ /[a-z]+_[a-z]+/)) {
 ////////////////////////////////////////////////////
 
 // Pipeline config
-ch_output_docs = file("$baseDir/docs/output.md", checkIfExists: true)
+ch_output_docs = file("${params.base_dir}/docs/output.md", checkIfExists: true)
```

Comment:
```diff
-$baseDir
+${params.base_dir}/
```

Modification to address the same issue as described [here](https://github.com/TheJacksonLaboratory/atacseq/pull/3/files#r419120532), when we cannot rely for local files from the cloned repo being available.